mkdocs.yml

```diff
@@ -42,7 +42,7 @@ plugins:
           show_symbol_type_heading: true
           docstring_style: numpy
           heading_level: 3
-    custom_templates: templates    - content.code.select
+    # custom_templates: templates    - content.code.select
```

Comment:
As far as I understand you managed to figure this out by `pip install mkdocstrings[python]`. Can you `rebase -i` or `git cherry-pick` this commit out? (_optional_)

Reply:
- [ ] rebase the commit out when i have nothing else to do